src/dependency-manager/src/schema/spec/resource-spec.ts

```diff
@@ -47,8 +47,8 @@ export class VolumeSpec {
   readonly?: boolean | string;
 }
 
-export type EnvironmentSpecValue = boolean | null | number | string;
-export const EnvironmentDictiory = DictionaryOfAny('boolean', 'null', 'number', 'string');
+export type EnvironmentSpecValue = boolean | null | number | object | string;
```

Comment:
Likewise, we should add `array` to the `EnvironmentSpecValue ` typescript type too.

Reply:
arrays are objects